.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+install:
+  - yarn install --frozen-lockfile
+script:
```

Comment:
Why not git hub actions?

Reply:
Travis was what I had within copy+paste reaching distance